playground/src/common/request.ts

```diff
@@ -35,12 +35,16 @@ export const apiGenAgoraData = async (config: GenAgoraDataConfig) => {
 
 export const apiStartService = async (config: StartRequestConfig): Promise<any> => {
   const url = `${REQUEST_URL}/start`
-  const { language, channel, userId, voiceType, graphName } = config
+  let { language, channel, userId, voiceType, graphName } = config
+
+  if (graphName == "camera.va.openai.azure.en" && language == "zh-CN") {
```

Comment:
this is too strange.

Reply:
If handled from the server side, it would lead to hard coding. The current design aims for the graph selection to be specified by the frontend.